### PR TITLE
Add License To VideoView

### LINKED ISSUE #78: Introduce display of license information on video playback
We need to introduce this, currently we have nothing. Some simple approach can be selected for now, adding it to the bottom of the description or something, it will later be designed properly, it was forgotten.

### PR DESCRIPTION
**Pull request by fcescob**
This PR fixes #78 ,  @bedeho and @blrhc please review if how it looks right now is good enough for Babylon.

### CODE REVIEW

File: src/api/schemas/extendedQueryNode.graphql [hunk 1]
```diff
@@ -63,28 +63,34 @@ enum VideoMediaEncoding {
...
 type UserDefinedLicense {
-  text: String!
+  content: String!
 }
 
-union License = UserDefinedLicense | CreativeCommonsLicense
+union License = UserDefinedLicense | KnownLicense
 
+type LicenseEntity {
+  id: ID!
```
Comment (kdembler) [inline src/api/schemas/extendedQueryNode.graphql:80]: Not sure if we should put all the fields here that the query node exposes. In general so far I've tried only including fields that we actually use. I guess things like `createdById` can be removed once the query node implementation changes so we shouldn't rely on them.

File: src/api/schemas/extendedQueryNode.graphql [hunk 2]
```diff
@@ -94,8 +100,7 @@ type VideoMedia {
   pixelWidth: Int!
 
   # Resolution height
-  pixelHeight: Int!
-
+  pixelHeight: Int
```
Comment (kdembler) [inline src/api/schemas/extendedQueryNode.graphql]: This type shouldn't be changed

File: src/mocking/data/raw/coverVideo.json [hunk 3]
```diff
@@ -22,5 +22,13 @@
     "handle": "SCHISM",
     "avatarPhotoUrl": "https://eu-central-1.linodeobjects.com/joystream/schismavatar.png",
     "coverPhotoUrl": null
+  },
+  "license": {
+    "id": "1503",
+    "attribution": null,
+    "type": {
+      "__typename": "UserDefinedLicense",
+      "content": "Free to consume and share, no derivatives or adaptations, copyright Jamie King 2020\n"
```
Comment (kdembler) [inline src/mocking/data/raw/coverVideo.json:31]: Should we include `\n` at the end?

File: src/views/VideoView/VideoView.tsx [hunk 4]
```diff
@@ -99,6 +102,23 @@ const VideoView: React.FC<RouteComponentProps> = () => {
...
         </DescriptionContainer>
+        <LicenseContainer>
+          {data?.video ? (
+            <>
+              <p>
+                License:{' '}
+                {data.video.license.type.__typename === 'KnownLicense' ? (
+                  <a href={data.video.license.type.url || ''}>{`${data.video.license.type.code}`}</a>
+                ) : (
+                  `${data.video.license.type.content}`
```
Comment (kdembler) [inline src/views/VideoView/VideoView.tsx]: I don't think we need that interpolation

File: src/views/VideoView/VideoView.tsx [hunk 5]
```diff
@@ -99,6 +102,23 @@ const VideoView: React.FC<RouteComponentProps> = () => {
             </>
           )}
         </DescriptionContainer>
+        <LicenseContainer>
+          {data?.video ? (
+            <>
+              <p>
+                License:{' '}
+                {data.video.license.type.__typename === 'KnownLicense' ? (
+                  <a href={data.video.license.type.url || ''}>{`${data.video.license.type.code}`}</a>
```
Comment (kdembler) [inline src/views/VideoView/VideoView.tsx]: The link should have `target="_blank"` and that should also imply `rel="noreferrer noopener"`

**Review by kdembler — CHANGES_REQUESTED** [review]

File: src/api/schemas/extendedQueryNode.graphql (hunk 1 above)
Comment (fcescob) [reply src/api/schemas/extendedQueryNode.graphql:80]: Good point, I will leave only the fields that we actually use

File: src/api/schemas/extendedQueryNode.graphql (hunk 2 above)
Comment (fcescob) [reply src/api/schemas/extendedQueryNode.graphql]: No idea how that happened, will revert the change

File: src/mocking/data/raw/coverVideo.json (hunk 3 above)
Comment (fcescob) [reply src/mocking/data/raw/coverVideo.json:31]: I just took the response from the query node for the production version of the cover video, I say let's leave it in for consistency even if its not needed.

File: src/views/VideoView/VideoView.tsx (hunk 5 above)
Comment (fcescob) [reply src/views/VideoView/VideoView.tsx]: Good point, changed that.

File: src/views/VideoView/VideoView.tsx (hunk 4 above)
Comment (fcescob) [reply src/views/VideoView/VideoView.tsx]: good catch, removed it.

File: src/mocking/data/raw/coverVideo.json (hunk 3 above)
Comment (kdembler) [reply src/mocking/data/raw/coverVideo.json:31]: Ok

File: src/views/VideoView/VideoView.tsx [hunk 6]
```diff
@@ -99,6 +100,27 @@ const VideoView: React.FC<RouteComponentProps> = () => {
...
+          {data?.video ? (
+            <>
+              <p>
+                License:{' '}
+                {data.video.license.type.__typename === 'KnownLicense' ? (
+                  <a
+                    href={data.video.license.type.url || ''}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >{`${data.video.license.type.code}`}</a>
```
Comment (kdembler) [inline src/views/VideoView/VideoView.tsx]: We don't need that interpolation as well, missed that before

**Review by kdembler — APPROVED** [review]

**Pull request merged**